hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/FlinkTaskContextSupplier.java

```diff
@@ -62,4 +64,9 @@ public RuntimeContext getFlinkRuntimeContext() {
     return Option.empty();
   }
 
+  @Override
+  public String getPartitionColumns(Properties props) {
```

Comment:
I do't think this belongs to the taskContextsupplier. This is very much product specific. Task Context supplier is meant to hold information about spark tasks and nothing more. Let's see if we can move elsewhere.